examples/umami/CommunityToolkit.Aspire.Hosting.Umami.AppHost/AppHost.cs

```diff
@@ -0,0 +1,17 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+var password = builder.AddParameter("db-password", "12345678");
+
+var postgres = builder
+    .AddPostgres("postgres", password: password, port: 61118)
+    .WithLifetime(ContainerLifetime.Persistent);
+var postgresdb = postgres.AddDatabase("postgresdb");
+
+var umami = builder
+    .AddUmami("umami", port: 55932)
+    .WithStorageBackend(postgresdb);
+
+var front = builder
+    .AddJavaScriptApp("front", "../umami-vite-app");
```

Comment:
I feel like there needs to be some kind of relationship here so that the frontend app is able to get the access key and endpoint for the backend rather than having them hard coded in the HTML.

It also would be nice to have a way to provide the website endpoint to the unami resource, but reading their docs that doesn't look possible.

Reply:
I first need to create a tracking website from Umami. So that would be the other way around. I will try to add a `WaitFor` and a custom logic for the JS app to register to umami dynamically.

Reply:
Done. It's a lot of boilerplate code though. I don't know if we should refactor it and expose it to be used by other developers or if we should leave it as is.